pkg/kubelet/status/manager_test.go

```diff
@@ -64,7 +65,8 @@ func (m *manager) testSyncBatch() {
 
 func newTestManager(kubeClient client.Interface) *manager {
 	podManager := kubepod.NewBasicPodManager(kubepod.NewFakeMirrorClient())
-	podManager.AddPod(testPod)
```

Comment:
Yeah, I've added a TODO about that. This PR mainly aims at fixing the bug. I'll send a next PR to refactor the test with a `newTestPod()`, :)